src/glyph/tests.rs

```diff
@@ -27,9 +27,33 @@ fn serialize_empty_glyph() {
     );
 }
 
+#[test]
```

Comment:
You may want to include the test cases I wrote in #237.

Reply:
those test cases don't really translate to this world where we don't hold on to the version after parsing.